src/Frozennode/Administrator/DataTable/Columns/Relationships/BelongsTo.php

```diff
@@ -29,8 +29,8 @@ public function build()
         $this->tablePrefix = $this->db->getTablePrefix();
         $nested            = $this->getNestedRelationships($options['relationship']);
 
-        $relevantName      = $nested['pieces'][sizeof($nested['pieces']) - 1];
-        $relevantModel     = $nested['models'][sizeof($nested['models']) - 2];
+        $relevantName      = $nested['pieces'][!empty($nested['pieces']) - 1];
```

Comment:
i don't think it can simpley replace `sizeof` with `!empty` here, it need a number to calculate, not a boolean value.